pkg/aws/policies.go

```diff
@@ -1042,16 +1042,14 @@ func (c *awsClient) DeleteOperatorRole(roleName string, managedPolicies bool) er
 func (c *awsClient) DeleteRole(role string) error {
 	_, err := c.iamClient.DeleteRole(&iam.DeleteRoleInput{RoleName: aws.String(role)})
 	if err != nil {
-		if err != nil {
-			if aerr, ok := err.(awserr.Error); ok {
-				switch aerr.Code() {
-				case iam.ErrCodeNoSuchEntityException:
-					return fmt.Errorf("operator role '%s' does not exists.skipping",
-						role)
-				}
+		if aerr, ok := err.(awserr.Error); ok {
+			switch aerr.Code() {
+			case iam.ErrCodeNoSuchEntityException:
+				return fmt.Errorf("operator role '%s' does not exist. Skipping",
+					role)
 			}
```

Comment:
Using a switch statement with only one case can sometimes be seen as unnecessary and may cause confusion for other developers.
What's the reason for using a switch here?

Reply:
This was existent prior to the changes I made, I assume it might've been copied over from a different function that required a switch statement, I can change it to use if statements.